examples/multisig-example/src/main/kotlin/network/finschia/sdk/example/Client.kt

```diff
@@ -237,4 +310,57 @@ suspend fun main() {
     //-----------------------------------------
     val result = client.broadcastTx(signedTx)
     println("result: $result")
+    multiSigAccSeq++
```

Comment:
```suggestion

    //-----------------------------------------
    // `MsgDelegate` scenario example
    //-----------------------------------------
    multiSigAccSeq++
```
How about adding this comment that indicates another scenario(MsgDelegate) starts.

Reply:
Thank you so much. I've updated it following your suggestion.